src/fiskaltrust.ifPOS/v1/de/Models/StartExportSessionByTransactionRequest.cs

```diff
@@ -6,9 +6,12 @@ namespace fiskaltrust.ifPOS.v1.de
     public class StartExportSessionByTransactionRequest
     {
         [DataMember(Order = 10)]
-        public ulong From { get; set; }
+        public string ClientId { get; set; } = string.Empty;
```

Comment:
What's the reason for using this default value here?

Reply:
no reason, i will remove it.
as trials shown, null works fine for our major supported tse-devices.
thx.